components/Tag/Tag.tsx

```diff
@@ -0,0 +1,32 @@
+import XMark from "../XMark/XMark";
+
+export type TagProps = {
+  label: string;
+  onClick: (event: React.MouseEvent<HTMLButtonElement>) => void;
+};
+
+export const Tag = ({ label, onClick }: TagProps) => {
+  if (!label) {
+    return null;
```

Comment:
Por otro lado, que te parece si en vez de retornar null, hacemos que aparezca un error así:
`throw new Error('Label cannot be an empty string');`
![Screen Shot 2023-01-24 at 10 05 42](https://user-images.githubusercontent.com/8450941/214299227-e6f8bab6-85eb-488d-8500-9c666e09584b.png)

Reply:
Ahi lo agregué!

Reply:
@nykka esto podria romper la web si por cualquier motivo se le pasa un `string` vacio no?, desde Sanity por ejemplo. 
Se podria agregar un Error Boundary para capturar ese error o retornar algun otro tipo de `div` o `span` con un mensaje de error que aparezca en la UI, o directamente no renderizarlo

Me inquieta retonar un error, pero no se jajaj

Reply:
Si, incluso cuando se corre `yarn test`, ese error termina generando un choclazo de texto enorme que puede llegar a ser molesto. Esto no encontré como sacarlo

Reply:
Ok, entonces dejemoslo como `null` y veamos que podemos hacer que Sanity. Gracias a ambos! <3